api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java

```diff
@@ -118,11 +119,9 @@
 @DirtiesContext(classMode = ClassMode.BEFORE_EACH_TEST_METHOD)
 @Transactional(propagation = Propagation.NOT_SUPPORTED)
 public class RuntimeControllerTest {
-
-  private static final String BILLING_PROJECT_ID = "aou-rw-1234";
```

Comment:
I am trying to remove BILLING_PROJECT_ID from this test, and use `WORKSPACE_NS` if that is namespace, use `GOOGLE_PROJECT_ID` if that is GCP project